### DISCUSSION TITLE
Uno CI/CD with MS App Center

### DISCUSSION THREAD

**Discussion by rizaanlakay** [unoplatform/uno#8549] [General]
We are looking to use Microsoft App Center to test and deploy our Android and IOS apps. We would like to use Xamarin test cloud too. Can MS App Center Build Android and IOS apps built with Uno?

**Comment by jeromelaban** [comment]
Yes, it's definitely possible. Uno apps are Xamarin apps, which means that all the documentation that you'll find to publish Xamarin apps applies directly to Uno apps.

**Reply by rizaanlakay** [reply]
Thanks Jerome
